### PR TITLE
Refactor/project

### PR DESCRIPTION
**Pull request by Delyc**
###  PR DESCRIPTION

- This PR creates the project section on all screens

### TASKS COMPLETED
- [x] Added the header content on behalf of the figma original design
- [x] Imported the Link and the Svg reusable component
- [x] Reused them to add the arrowed links
- [x] Implemented a sample image to match the design

### HOW CAN THIS PR BE TESTED

```
git clone https://github.com/TheGymRwanda/unit8-maroon
git checkout ft/project
npm install
npm run dev

```

### BEFORE SUBMITTING YOUR PR, ENSURE IT HAS THE FOLLOWING

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### TICKET NUMBER

#014: https://www.notion.so/apeunit/Projects-section-4ce681c3ee9d44529d7c4543172be7f4

### CODE REVIEW

**Review by Nkaka23dev — APPROVED** [review]
Code Looks fine so far.

**Review by haroldmud — APPROVED** [review]

**Review by haroldmud — APPROVED** [review]

File: components/section/Project.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "Bitlipa",
+    "Symplifi",
+    "UTU.IO",
+    "NeueUX",
+    "Lab3",
+    "Even"
+  ];
+
+  const images = [
+    { image: "/assets/images/project-sample.png", text: "Peer-to-Peer learning platform" },
```
Comment (ishoborabyose) [inline components/section/Project.jsx]: Looks good, but I think you should convert images from png to webp.

File: components/section/Project.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    { image: "/assets/images/project-sample.png", text: "Peer-to-Peer learning platform" },
```
Comment (ishoborabyose) [inline components/section/Project.jsx]: I think you should convert images from png to webp.

File: components/section/Project.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    { image: "/assets/images/project-sample.png", text: "Peer-to-Peer learning platform" },
```
Comment (ishoborabyose) [inline components/section/Project.jsx]: I think you should convert images from png to webp.

File: components/section/Project.jsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    { image: "/assets/images/project-sample.png", text: "Peer-to-Peer learning platform" },
```
Comment (ishoborabyose) [inline components/section/Project.jsx]: I think you should convert images from png to webp.

File: components/section/Project.jsx [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    { image: "/assets/images/project-sample.png", text: "Peer-to-Peer learning platform" },
```
Comment (ishoborabyose) [inline components/section/Project.jsx]: I think you should convert images from png to webp.

File: components/section/Project.jsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    { image: "/assets/images/project-sample.png", text: "Peer-to-Peer learning platform" },
```
Comment (ishoborabyose) [inline components/section/Project.jsx]: I think you should convert images from png to webp.

File: components/section/Project.jsx [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    { image: "/assets/images/project-sample.png", text: "Peer-to-Peer learning platform" },
```
Comment (ishoborabyose) [inline components/section/Project.jsx]: I think you should convert images from png to webp.

**Review by ishoborabyose — CHANGES_REQUESTED** [review]

File: components/section/Project.jsx [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    "Dacade",
+    "Bitlipa",
+    "Symplifi",
+    "UTU.IO",
+    "NeueUX",
+    "Lab3",
+    "Even"
+  ];
+
+  const images = [
```
Comment (ishoborabyose) [inline components/section/Project.jsx:18]: I think you should use next image component

**Review by ishoborabyose — CHANGES_REQUESTED** [review]

**Review by ishoborabyose — APPROVED** [review]

**Pull request merged**